.github/workflows/master.yml

```diff
@@ -53,9 +53,9 @@ jobs:
           - '3.3'
           - '3.4'
           - '3.5'
-        spark-archive: [""]
-        exclude-tags: [""]
-        comment: ["normal"]
+        spark-archive: [ "" ]
+        exclude-tags: [ "" ]
+        comment: [ "normal" ]
```

Comment:
please avoid such unnecessary changes.

Reply:
Due to the 'Action on Save' tool reformat the yml code. I have reverted the cord.